README.md

```diff
@@ -40,6 +40,10 @@ Jaeger is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CN
   * [Kubernetes templates](https://github.com/jaegertracing/jaeger-kubernetes)
   * [OpenShift templates](https://github.com/jaegertracing/jaeger-openshift)
 
+The Jaeger contributors meet bi-weekly, and everyone is welcome to join.
+* When: 1st and 3rd Fridays of the month, [4pm UTC](https://www.google.de/search?q=4pm+utc)
+* Where: https://uber.zoom.us/j/680943508
```

Comment:
I suggest just adding a link to the doc, which has all the information (so that we don't have to duplicate it).

https://docs.google.com/document/d/1ZuBAwTJvQN7xkWVvEFXj5WU9_JmS5TPiNbxCJSvPqX0/